backend/src/test/java/ca/gov/dtsstn/cdcp/api/data/repository/ConfirmationCodeRepositoryIT.java

```diff
@@ -0,0 +1,39 @@
+package ca.gov.dtsstn.cdcp.api.data.repository;
+import static org.assertj.core.api.Assertions.assertThat;
+import java.time.Instant;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase.Replace;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.context.annotation.Import;
+import org.springframework.test.context.ActiveProfiles;
+import ca.gov.dtsstn.cdcp.api.config.DataSourceConfig;
+
+
+@DataJpaTest
+@ActiveProfiles("test")
+@Import({ DataSourceConfig.class })
+@AutoConfigureTestDatabase(replace = Replace.NONE)
+public class ConfirmationCodeRepositoryIT {
+
+	@Autowired ConfirmationCodeRepository confirmationCodeRepository;
+
+	@Test
+	@DisplayName("Test confirmationCodeRepository.findBy(..)")
+	void testFindById() {
+		assertThat(confirmationCodeRepository.findAll()).isNotEmpty();
+		assertThat(confirmationCodeRepository.findById("a6ea4925-f813-493e-80ec-a5b90ca28b6c")).isNotEmpty();
+		assertThat(confirmationCodeRepository.findById("bogusId")).isEmpty();
+	}
+
+	@Test
+	@DisplayName("Test confirmationCodeRepository.deleteByExpiredCode(..)")
+	void testDeleteByExpiredCode() {
+
```

Comment:
Test method name should match the method name being tested.

```suggestion
	@DisplayName("Test confirmationCodeRepository.deleteByExpiryDateLessThan(..)")
	void testDeleteByExpiryDateLessThan() {
```